WORKSPACE

```diff
@@ -29,21 +29,21 @@ load("//scala:scala_cross_version.bzl", "scala_mvn_artifact")
 # test adding a scala jar:
 maven_jar(
     name = "com_twitter__scalding_date",
-    artifact = scala_mvn_artifact("com.twitter:scalding-date:0.17.0"),
+    artifact = scala_mvn_artifact("com.twitter:scalding-date:0.17.0", "2.11"),
```

Comment:
thought (which can be deferred to later PR): 
WDYT about:
`artifact = scala_mvn_artifact("com.twitter:scalding-date:0.17.0", default_scala_major_version),`
where `default_scala_major_version` is defined as `extract_major_version_underscore(default_scala_version())`
Will ease changing the default version for rules_scala when we'll need it